store/copr/mpp.go

```diff
@@ -293,7 +346,7 @@ func (m *mppIterator) Close() error {
 	if atomic.CompareAndSwapUint32(&m.closed, 0, 1) {
 		close(m.finishCh)
 	}
-	m.rpcCancel.CancelAll()
+	m.cancelFunc()
```

Comment:
Test this logic: two root tasks , one returns error and the other hangs. Without kill, we can end the hanging goroutine by just closing the resources correctly.

Reply:
added